diff --git a/Makefile b/Makefile
index d186eb3..93cea41 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 TARGET      = devify
 
-PREFIX     ?= /usr/local
+PREFIX     ?= /usr/
 INSTALL     = install -Dm755
 RM         ?= rm -f
 
@@ -14,7 +14,7 @@ install: $(TARGET)
 	
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL) $(TARGET) $(DESTINATIONBIN)$(TARGET)
-	install -Dm644 devify.rules $(DESTINATIONUDEV)99-devify.rules
+	install -Dm644 devify.rules $(DESTINATIONUDEV)60-devify.rules
 	mkdir -p $(DESTDIR)$(PREFIX)/share/icons
 	install -d $(DESTDIR)$(PREFIX)/share/icons/MonoDev
 	install -Cm644 icons/* $(DESTDIR)$(PREFIX)/share/icons/MonoDev
@@ -23,7 +23,7 @@ uninstall: $(TARGET)
 
 	rm $(DESTINATIONBIN)$(TARGET)
 	rm -rf $(DESTDIR)$(PREFIX)/share/icons/MonoDev
-	rm $(DESTINATIONUDEV)99-devify.rules
+	rm $(DESTINATIONUDEV)60-devify.rules
 
 clean:
 	$(RM) ./devify
diff --git a/devify b/devify
index 0e16e14..6419515 100755
--- a/devify
+++ b/devify
@@ -1,13 +1,14 @@
 #!/usr/bin/env bash
 
 status=$2
-icon_pack=/usr/local/share/icons/MonoDev
+icon_pack=/usr/share/icons/MonoDev
 user=$(who | head -n 1 | awk '{print $1}')
 wal_dir=/home/$user/.cache/wal/colors
 icon_color=$(grep -oP '#.*([A-Za-z]|[0-9])' $icon_pack/main_color.svg)
 
-DBUS_SESSION_BUS_ADDRESS=unix:path="/run/user/$( id -u "$user" )/bus"
-export DBUS_SESSION_BUS_ADDRESS
+# DBUS_SESSION_BUS_ADDRESS=unix:path="/run/user/$( id -u "$user" )/bus"
+# export DBUS_SESSION_BUS_ADDRESS
+export DISPLAY=:0
 
 notify() {
   if [ "$status" == "add" ]; then
