diff --git a/devify b/devify
index e65d3a4..e8c701b 100755
--- a/devify
+++ b/devify
@@ -10,18 +10,18 @@ else
 fi
 
 status=$2
-icon_pack=/usr/local/share/icons/MonoDev
-sound_pack=/usr/local/share/sounds/MonoDev
+icon_pack=/usr/share/icons/MonoDev
+sound_pack=/usr/share/sounds/MonoDev
 user=$(who | head -n 1 | awk '{print $1}')
 uid=$( id -u "$user" )
 wal_dir=/home/$user/.cache/wal/colors
 icon_color=$(grep -oP '#.*([A-Za-z]|[0-9])' $icon_pack/main_color.svg)
 POWER_ADAPTER_STATUS="/var/run/devify_power_adapter_status"
 
-DBUS_SESSION_BUS_ADDRESS=unix:path="/run/user/$uid/bus"
-export DBUS_SESSION_BUS_ADDRESS
+# DBUS_SESSION_BUS_ADDRESS=unix:path="/run/user/$uid/bus"
+# export DBUS_SESSION_BUS_ADDRESS
 # required by Libcanberra
-XDG_RUNTIME_DIR=/run/user/$uid
+# XDG_RUNTIME_DIR=/run/user/$uid
 export XDG_RUNTIME_DIR
 export DISPLAY=:0
 export WAYLAND_DISPLAY=wayland-0
