diff --git a/QLog.pro b/QLog.pro
index ce918fe..3619792 100644
--- a/QLog.pro
+++ b/QLog.pro
@@ -462,7 +462,7 @@ unix:!macx {
    INSTALLS += target desktop icon metainfo

    INCLUDEPATH += /usr/local/include
-   LIBS += -L/usr/local/lib -lhamlib -lsqlite3
+   LIBS += -L/usr/lib64/hamlib -L/usr/lib64 -lhamlib -lsqlite3
    equals(QT_MAJOR_VERSION, 6): LIBS += -lqt6keychain
    equals(QT_MAJOR_VERSION, 5): LIBS += -lqt5keychain
 }
